addons/mrp_account/data/mrp_account_demo.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
no need the <data>, you can put the noupdate in <odoo 
see https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#format : "The tag <data> is only used to set not-updatable data with noupdate=1. If there is only not-updatable data in the file, the noupdate=1 can be set on the <odoo> tag and do not set a <data> tag."